storyscript/Api.py

```diff
@@ -11,18 +11,18 @@ class StoryscriptCompilationResult:
     Contains the compiled story or a list of compilation errors.
     """
 
-    def __init__(self, result, errors):
+    def __init__(self, result, errors, deprecations=[]):
```

Comment:
```suggestion
    def __init__(self, result, errors, deprecations):
```

This will require us to check all spots where this change is required. It's okay, because it's an internal API.